cmd/vulcan-trivy/vulcan-trivy.go

```diff
@@ -49,23 +44,34 @@ type Results struct {
 
 type ScanResponse []struct {
 	Target          string `json:"Target"`
+	Class           string `json:"Class"`
+	Type            string `json:"Type"`
 	Vulnerabilities []struct {
 		VulnerabilityID  string   `json:"VulnerabilityID"`
 		PkgName          string   `json:"PkgName"`
+		PkgPath          string   `json:"PkgPath"`
 		InstalledVersion string   `json:"InstalledVersion"`
 		FixedVersion     string   `json:"FixedVersion"`
 		Title            string   `json:"Title,omitempty"`
 		Description      string   `json:"Description,omitempty"`
```

Comment:
We are not using Title / Description. It adds detailed information but it could be too much info.

```json
{
  "Title": "zlib: A flaw in zlib-1.2.11 when compressing (not decompressing!) certain inputs.",
  "Description": "zlib before 1.2.12 allows memory corruption when deflating (i.e., when compressing) if the input has many distant matches."}
```

Reply:
I think is ok to have these attributes mapped at check level even if we not use them for now.
We just need to pay attention if we decide to use them in the future in terms of vulnerability size.